### PR TITLE
Added spoken form to API

### PR DESCRIPTION
**Pull request by AndreasArvidsson**

### CODE REVIEW

File: src/command.py [hunk 1]
```diff
@@ -68,8 +77,10 @@ def cursorless_multiple_target_command(
     ):
         """Execute multi-target cursorless command"""
         args = list(filter(lambda x: x is not NotSet, [arg1, arg2, arg3]))
+        spoken_form = " ".join(last_phrase["phrase"])
```
Comment (pokey) [inline src/command.py:80]: Just to check, if I say "disk take air", this won't pick up the initial "disk" right?

Comment (AndreasArvidsson) [reply src/command.py:80]: It will. So we are going to have some discipline when we do the test cases.

Comment (pokey) [reply src/command.py:80]: Hmm. Ok I'm gonna approve this one just so you're not blocked on adding tests to your other PRs but not sure I'm in love with passing through the whole command to every cursorless command. Let's discuss later

**Review by pokey — APPROVED** [review]

File: src/command.py (hunk 1 above)
Comment (AndreasArvidsson) [reply src/command.py:80]: Yes I think we could improve this. Just unsure of the best solution right now.

**Pull request merged**